### ISSUE TITLE
AUR package broken?

### ISSUE THREAD

**Issue by nwg-piotr** [ErikReider/SwayNotificationCenter#135]
```txt
Packages (1) swaync-0.6.2-1 

:: Proceed with package upgrade? [Y/n] 
:: Cloning swaync package repo...
Cloning into 'swaync'...
remote: Enumerating objects: 36, done.
remote: Counting objects: 100% (36/36), done.
remote: Compressing objects: 100% (34/34), done.
remote: Total 36 (delta 16), reused 18 (delta 2), pack-reused 0
Receiving objects: 100% (36/36), 4.81 KiB | 4.81 MiB/s, done.
Resolving deltas: 100% (16/16), done.
==> Making package: swaync 0.6.2-1 (Sat Jun 25 17:31:23 2022)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading SwayNotificationCenter-0.6.2.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 3067k    0 3067k    0     0  3740k      0 --:--:-- --:--:-- --:--:-- 9848k
==> Validating source files with sha256sums...
    SwayNotificationCenter-0.6.2.tar.gz ... FAILED
==> ERROR: One or more files did not pass the validity check!
```

**Comment by ErikReider** [comment]
Forgot to update the checksum...

**Comment by ErikReider** [comment]
Fixed [a932273](https://github.com/ErikReider/SwayNotificationCenter/commit/a93227310131e23f5e7fd8b81f257775609ee7ca)
